website/source/guides/load-balancing/fabio.html.md

```diff
@@ -0,0 +1,233 @@
+
+We can now register our job for Apache:
+
+```shell
+$ nomad job run webserver.nomad 
+==> Monitoring evaluation "c7bcaf40"
+    Evaluation triggered by job "webserver"
+    Evaluation within deployment: "e3603b50"
+    Allocation "20951ad4" created: node "510898b6", group "webserver"
+    Allocation "43807686" created: node "28d7f859", group "webserver"
+    Allocation "7b60eb24" created: node "f3739267", group "webserver"
+    Evaluation status changed: "pending" -> "complete"
+==> Evaluation "c7bcaf40" finished with status "complete"
+```
+You have now deployed and registered your web servers with fabio! At this point,
+you should be able to visit any of the Nomad clients at port `9999` and
+see the default web page for Apache web server. If you visit fabio's web
+interface by going to any of the client nodes at port `9998`, you will see that
+the routing table has been populated as shown below (**Note:** your destination IP
+addresses will be different).
+
+[![Routing Table][routing-table]][routing-table]
+
+Feel free to reduce the `count` in `webserver.nomad` for testing purposes. You
+will see that you still get routed to the Apache home page by accessing
+any client node on port `9999`. Accordingly, the routing table
+in the web interface on port `9999` will reflect the changes.
+
+### Step 5: Place Nomad Client Nodes Behing AWS Load Balancer
+
+At this point, you are ready to place your Nomad client nodes behind an AWS load
+balancer. Your Nomad client nodes may change over time, and it is important
+to provide your end users with a single endpoint to access your services. This guide will use the [Classic Load Balancer][classic-lb].
+
+The AWS [documentation][classic-lb-doc] provides instruction on how to create a
+load balancer. The basic steps involve creating a load balancer, registering
+instances behind the load balancer (in our case these will be the Nomad client
+nodes), creating listeners, and configuring health checks.
+
+Once you are done
+with this, you should be able to hit the DNS name of your load balancer at port
+80 (or whichever port you configured in your listener) and see the home page of
+Apache web server. If you configured your listener to also foward traffic to
+the web interface at port `9998`, you should be able to access that as well.
+
+[![Home Page][lb-homepage]][lb-homepage]
+
+[![Routing Table][lb-routing-table]][lb-routing-table]
+
+[apache]: https://httpd.apache.org/
+[classic-lb]: https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/introduction.html
+[classic-lb-doc]: https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/elb-getting-started.html
+[elb]: https://aws.amazon.com/elasticloadbalancing/
+[fabio]: https://fabiolb.net/
+[lb-homepage]: /assets/images/lb-homepage.png
+[lb-routing-table]: /assets/images/lb-routing-table.png
+[routing-table]: /assets/images/routing-table.png
+[system]:https://www.nomadproject.io/docs/runtime/schedulers.html#system
```

Comment:
Broken link .. should be `/docs/schedulers.html#system`. Also, you can use a relative link for this.